src/zrc/datamodel/models.py

```diff
@@ -294,6 +446,26 @@ class Meta:
         verbose_name = 'zaakobject'
         verbose_name_plural = 'zaakobjecten'
 
+    def _get_object(self) -> dict:
+        """
+        Retrieve the `Object` specified as URL in `ZaakObject.object`.
+
+        :return: A `dict` representing the object.
+        """
+        if not hasattr(self, '_object'):
+            object_url = self.object
+            self._object = None
+            if object_url:
+                Client = import_string(settings.ZDS_CLIENT_CLASS)
+                client = Client.from_url(object_url)
+                client.auth = APICredential.get_auth(
+                    object_url,
+                    scopes=['zds.scopes.zaaktypes.lezen']
```

Comment:
don't think there's any point in passing a ZRC scope

Reply:
You are right. I'm not sure how this should be tackled on a general level.